robot.py

```diff
@@ -145,20 +146,26 @@ def teleopPeriodic(self):
         joystick_vz = -rescale_js(self.joystick.getZ(), deadzone=0.2, exponential=20.0, rate=self.spin_rate)
         joystick_hat = self.joystick.getPOV()
 
-        gamepad_vx = rescale_js(self.gamepad.getX(10), deadzone=0.01, exponential=0.5, rate=1)
-        gamepad_vy = rescale_js(self.gamepad.getY(10), deadzone=0.01, exponential=0.5, rate=1)
+        gamepad_vx = -rescale_js(self.gamepad.getY(10), deadzone=0.1, exponential=1.5, rate=0.5)
+        gamepad_vy = -rescale_js(self.gamepad.getX(10), deadzone=0.1, exponential=1.5, rate=0.5)
         # TODO Tune these terms for the gamepad.
 
         if joystick_vx or joystick_vy or joystick_vz:
+            if self.joystick.getRawButton(6):
+                self.chassis.field_oriented = False
+            else:
+                self.chassis.field_oriented = True
             self.chassis.set_inputs(joystick_vx, joystick_vy, joystick_vz)
-        elif gamepad_vx or gamepad_vy:
+        elif (gamepad_vx or gamepad_vy) and self.gamepad.getRawButton(9):
```

Comment:
Wouldn't it be more prudent to just check the button is pressed? Use its name too.